modules/apps/subscription/subscription-service/src/main/java/com/liferay/subscription/internal/messaging/SubscriptionSenderMessageListener.java

```diff
@@ -28,6 +37,27 @@
 )
 public class SubscriptionSenderMessageListener extends BaseMessageListener {
 
+	@Activate
+	protected void activate(BundleContext bundleContext) {
+		DestinationConfiguration destinationConfiguration =
```

Comment:
@JanisZhang Please inline this variable, also in https://github.com/liferay-core-infra/liferay-portal/pull/6108 and https://github.com/liferay-core-infra/liferay-portal/pull/6107